spec/rubocop/cop/style/super_arguments_spec.rb

```diff
@@ -0,0 +1,220 @@
+          super
+        end
+      RUBY
+    end
+  end
+
+  shared_examples 'no offense' do |description, args, forwarded_args = args|
+    it "registers no offense when using def `#{description} (#{args}) => (#{forwarded_args})`" do
+      expect_no_offenses(<<~RUBY)
+        def method(#{args})
+          super(#{forwarded_args})
+        end
+      RUBY
+    end
+
+    it "registers no offense when using defs `#{description} (#{args}) => (#{forwarded_args})`" do
+      expect_no_offenses(<<~RUBY)
+        def self.method(#{args})
+          super(#{forwarded_args})
+        end
+      RUBY
+    end
+  end
+
+  it_behaves_like 'offense', 'no arguments', ''
+  it_behaves_like 'offense', 'single positional argument', 'a'
+  it_behaves_like 'offense', 'multiple positional arguments', 'a, b'
+  it_behaves_like 'offense', 'multiple positional arguments with default', 'a, b, c = 1', 'a, b, c'
+  it_behaves_like 'offense', 'positional/keyword argument', 'a, b:', 'a, b: b'
+  it_behaves_like 'offense', 'positional/keyword argument with default', 'a, b: 1', 'a, b: b'
+  it_behaves_like 'offense', 'positional/keyword argument both with default', 'a = 1, b: 2', 'a, b: b'
+  it_behaves_like 'offense', 'named block argument', '&blk'
+  it_behaves_like 'offense', 'positional splat arguments', '*args'
+  it_behaves_like 'offense', 'keyword splat arguments', '**kwargs'
+  it_behaves_like 'offense', 'positional/keyword splat arguments', '*args, **kwargs'
+  it_behaves_like 'offense', 'positionalkeyword splat arguments with block', '*args, **kwargs, &blk'
+  it_behaves_like 'offense', 'keyword arguments mixed with forwarding', 'a:, **kwargs', 'a: a, **kwargs'
+  it_behaves_like 'offense', 'tripple dot forwarding', '...'
+  it_behaves_like 'offense', 'tripple dot forwarding with extra arg', 'a, ...'
+
+  it_behaves_like 'no offense', 'different amount of positional arguments', 'a, b', 'a'
+  it_behaves_like 'no offense', 'positional arguments in different order', 'a, b', 'b, a'
+  it_behaves_like 'no offense', 'keyword arguments in different order', 'a:, b:', 'b: b, a: a'
+  it_behaves_like 'no offense', 'positional/keyword argument mixing', 'a, b', 'a, b: b'
+  it_behaves_like 'no offense', 'positional/keyword argument mixing reversed', 'a, b:', 'a, b'
+  it_behaves_like 'no offense', 'block argument with different name', '&blk', '&other_blk'
+  it_behaves_like 'no offense', 'keyword arguments and hash', 'a:', '{ a: a }'
+  it_behaves_like 'no offense', 'keyword arguments with send node', 'a:, b:', 'a: a, b: c'
+  it_behaves_like 'no offense', 'tripple dot forwarding with extra param', '...', 'a, ...'
+  it_behaves_like 'no offense', 'tripple dot forwarding with different param', 'a, ...', 'b, ...'
+  it_behaves_like 'no offense', 'keyword forwarding with extra keyword', 'a, **kwargs', 'a: a, **kwargs'
+
+  context 'Ruby >= 3.1', :ruby31 do
+    it_behaves_like 'offense', 'hash value omission', 'a:'
+    it_behaves_like 'offense', 'anon block forwarding', '&'
```

Comment:
Can you also use "anonymous" for the terminology in the test code, just like in the product code?
https://github.com/rubocop/rubocop/pull/12427#discussion_r1610340732

```suggestion
    it_behaves_like 'offense', 'anonymous block forwarding', '&'
```